docs/resources/dws_cluster.md

```diff
@@ -113,6 +113,8 @@ The following arguments are supported:
 * `keep_last_manual_snapshot` - (Optional, Int) The number of latest manual snapshots that need to be
   retained when deleting the cluster.
 
+* `elb_id` - (Optional, String) The ELB ID.
```

Comment:
what the ELB resource is this parameter match?
loadbalancer? listener? or pool?

Reply:
fixed. This parameter indicate load balancer.